lms/models/register.py

```diff
@@ -0,0 +1,44 @@
+import re
+
+from flask import url_for
+from flask_babel import gettext as _  # type: ignore
+from flask_mail import Message  # type: ignore
+from itsdangerous import URLSafeTimedSerializer
+
+from lms.lmsdb.models import User
+from lms.lmsweb import config, webapp, webmail
+from lms.models.errors import EmptyPasswordError, UnhashedPasswordError
+
+
+SERIALIZER = URLSafeTimedSerializer(config.SECRET_KEY)
+
+
+def retrieve_salt(user: User) -> str:
+    try:
+        re.findall(r'\$(.*)\$', user.password)[0]
+    except IndexError:
+        if user.password.name:
+            raise UnhashedPasswordError
+        raise EmptyPasswordError
+
+
+def generate_confirmation_token(user: User) -> str:
+    return SERIALIZER.dumps(user.mail_address, salt=retrieve_salt(user))
+
+
+def send_confirmation_mail(user: User) -> None:
+    token = generate_confirmation_token(user)
+    subject = _('מייל אימות - %(site_name)s', site_name=config.SITE_NAME)
+    msg = Message(
+        subject, sender=f'lms@{config.MAILGUN_DOMAIN}',
```

Comment:
We should allow the user to choose the username (instead of `lms`)

Reply:
Added `MAIL_DEFAULT_SENDER` variable in the config so user will choose it